recipes/h5io/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "h5io" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "267af4fe374f287aa5e77ec386673ea22b02c6303c1485445a2b8c0a37b23f39" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy
+    - h5py
+  run:
+    - python
+    - numpy
+    - h5py
+
+test:
+  imports:
+    - h5io
+
+about:
+  home: http://github.com/h5io/h5io
+  license: BSD-3-Clause
+  license_family: BSD
+  summary: 'Python Objects Onto HDF5'
+  description: |
+    h5io is a package designed to facilitate saving some standard Python objects into the 
+    forward-compatible HDF5 format. It is a higher-level package than h5py.
+  doc_url: https://github.com/h5io/h5io
+  dev_url: https://github.com/h5io/h5io
+
+extra:
+  recipe-maintainers:
+    - jan-janssen
```

Comment:
@larsoner I meant adding you here.

Reply:
Yes I know, I'm asking a different question about removing the `example` file above...